carbonmark-api/src/utils/helpers/fetchMarketplaceListings.ts

```diff
@@ -11,28 +14,46 @@ type Params = {
   key: string; // Project key `"VCS-981"`
   vintage: string; // Vintage string `"2017"`
   fastify: FastifyInstance; // Fastify instance
+  network?: NetworkParam;
 };
 
 const filterUnsoldActivity = (activity: { activityType?: string }) =>
   activity.activityType !== "Sold";
 
+export const getCreditListings = async (params: {
```

Comment:
As with the other comments this is still retuning a Project right? can we call it `getCreditProjects`?